packages/media-downloader/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import {
+	getConfig,
+	getSQSClient,
+} from '@guardian/transcription-service-backend-common';
+import { Upload } from '@aws-sdk/lib-storage';
+import { S3Client } from '@aws-sdk/client-s3';
+import { createReadStream } from 'node:fs';
+import { getNextJob } from './sqs';
+import { downloadMedia, MediaMetadata } from './yt-dlp';
+
+const uploadToS3 = async (
+	s3Client: S3Client,
+	metadata: MediaMetadata,
+	bucket: string,
+) => {
+	const fileStream = createReadStream(`${metadata.mediaPath}`);
+	try {
+		const upload = new Upload({
+			client: s3Client,
+			params: {
+				Bucket: bucket,
+				Key: `downloaded-media/${metadata.title}.${metadata.extension}`,
```

Comment:
Seems possible that two different files will share a title. Could this result in files being accidentally overwritten or permissions leaking to the wrong user?

Reply:
apologies, this code shouldn't have been checked in, in the media-download version it just uses the `id`